packages/react-ui/src/hooks/useFilter.ts

```diff
@@ -0,0 +1,12 @@
+import { useState } from "react"
```

Comment:
we're going to want this to be useFilter**s** very soon - though it's ok if we leave as is for now and next PR we rename it